src/Transports.AspNetCore.SystemTextJson/GraphQLRequestDeserializer.cs

```diff
@@ -124,5 +118,17 @@ static bool DetermineTokenType(in ReadOnlySequence<byte> buffer, out JsonTokenTy
             Variables = fc.TryGetValue(GraphQLRequestBase.VariablesKey, out var variablesValue) ? variablesValue[0].ToDictionary() : null,
             OperationName = fc.TryGetValue(GraphQLRequestBase.OperationNameKey, out var operationNameValues) ? operationNameValues[0] : null
         };
+
+        /// <summary>
```

Comment:
You don't need this helper method. [CancellationToken.ThrowIfCancellationRequested](https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken.throwifcancellationrequested?view=netframework-4.8) throws a OperationCanceledException only if this token has had cancellation requested. It's functionality equivalent to 

`if (token.IsCancellationRequested) throw new OperationCanceledException(token);  `

Looking forward to this PR!

Reply:
Ahhh, I see. Silly me haha. Cheers!